boards/arm/96b_argonkey/doc/96b_argonkey.rst

```diff
@@ -0,0 +1,164 @@
+.. _96b_argonkey:
+
+96Boards Argonkey
+################
+
+Overview
+********
+
+96Boards Argonkey board is based on the STMicroelectronics STM32F412CG
+Cortex M4 CPU. Zephyr applications use the 96b_argonkey configuration
+to run on these boards.
+
+This board acts as a sensor hub platform for all 96Boards compliant
+family products. It can also be used as a standalone board.
+
+.. figure:: img/96b-argonkey-front.jpg
+     :width: 557px
+     :align: center
+     :height: 480px
+     :alt: 96Boards Argonkey
+
+     96Boards Argonkey
+
+Hardware
+********
+
+96Boards Argonkey provides the following hardware components:
+
+- STM32F412CG in UFQFPN48 package
+- ARM |reg| 32-bit Cortex |reg|-M4 CPU with FPU
+- 100 MHz max CPU frequency
+- 1.8V work voltage
+- 1024 KB Flash
+- 256 KB SRAM
+- On board sensors:
+
+  - Humidity: ST Micro HTS221
+  - Temperature/Pressure: ST Micro LPS22HB
+  - ALS: Intersil ISL29034
+  - Proximity: STMicro VL53L0X
+  - Acclerometer/Gyroscope: STMicro LSM6DSL
+  - Geomagnetic: STMicro LIS2MDL
+  - AMR Hall sensor: MRMS501A
+  - Microphone: STMicro MP34DT05
```

Comment:
You used sometime "ST Micro" or "STMicro", please align.

Reply:
STMicro (without a space) would be preferred

Reply:
ok